src/store/actions/test.js

```diff
@@ -32,5 +35,13 @@ describe( 'Store', () => {
 			expect( action.type ).toEqual( ActionTypes.BLOCK.DELETE );
 			expect( action.clientId ).toEqual( '1' );
 		} );
+
+		it( 'should create an action to create a block', () => {
+			registerCoreBlocks();
+			const newBlock = createBlock( 'core/code', { content: 'new test text for a core/code block' } );
+			const action = actions.createBlockAction( '1', newBlock );
+			expect( action.type ).toEqual( ActionTypes.BLOCK.CREATE );
+		} );
```

Comment:
Any particular reason for not testing the `clientId` value too as in the rest if this test case?

Reply:
addressed in 6ad3813 - and for completeness added check for the block in e556378